E2_ESRGAN/lib/dataset.py

```diff
@@ -24,6 +20,7 @@ def scale_fn(image, *args, **kwargs):
     low_resolution = tf.clip_by_value(low_resolution, 0, 255)
     high_resolution = tf.clip_by_value(high_resolution, 0, 255)
     return low_resolution, high_resolution
+  scale_fn.dimension = dimension
```

Comment:
Doesn't the usage of `dimension` work the way you used it before?

Reply:
well, if I assign this dimension, then I can fetch this attribute in augment_fn, else, I need to pass the dimension again. It's just a way to pass the same param again and again, into functions which will be actually chain mapped on the dataset.